changes/1267.bugfix.rst

```diff
@@ -0,0 +1 @@
+Briefcase's Progress Bar and Wait Bar will no longer be rendered if the ``FORCE_COLOR`` environment variable is set.
```

Comment:
```suggestion
The Progress Bar and Wait Bar will no longer be rendered if Briefcase runs on a non-TTY console (e.g., in CI environments, or when being piped through another process).
```

Reply:
I had to think to remember why I choose this wording...

So, Briefcase already doesn't render progress bars or the wait bar if `stdout` isn't attached to a TTY. AFAICT, it is only when `FORCE_COLOR` is set that Rich will start rendering these without a TTY....or `force_terminal=True` is explicitly specified when the `Console` is created.

Mentioning `FORCE_COLOR` in the RN does seem especially esoteric...but it's most accurate.

Reply:
Ah - got it. Makes sense.